citesphere/src/main/java/edu/asu/diging/citesphere/core/factory/zotero/impl/ItemFactory.java

```diff
@@ -65,22 +65,36 @@ public Item createItem(ICitation citation) {
         
         data.setCreators(new ArrayList<>());
         citation.getAuthors().forEach(a -> {
-            Creator creator = new Creator();
-            creator.setFirstName(a.getFirstName());
-            creator.setLastName(a.getLastName());
-            creator.setCreatorType(ZoteroConstants.CREATOR_TYPE_AUTHOR);
-            data.getCreators().add(creator);
+            if(a.getLastName() !=null) {
```

Comment:
condition should check last name or first name. Would madonna be the last name or first name? Just because someone put it in the first name field, doesn't mean it should not be created.